benchmarks/tests/bellman_ford_mod.py

```diff
@@ -53,19 +53,31 @@ def bellman_ford_test ( ):
 
     from numpy import array
     from numpy import zeros
+    from numpy import cos
 
-    e_num = 10
-    v_num = 6
+    e_num = 19900
+    v_num = 200
 
-    e = array( (( 1, 4, 1, 2, 4, 2, 5, 3, 5, 3 ), \
-                ( 0, 1, 2, 4, 0, 5, 0, 2, 3, 0 )) )
+    e = zeros ( (2, e_num), dtype = int )
+    weights = zeros (e_num, dtype = float )
+    idx = 0
 
-    e_weight = array( (-3.0,  6.0, -4.0, -1.0,  4.0, \
-                       -2.0,  2.0,  8.0, -3.0,  3.0 ) )
+    for i in  range( v_num ):
+        for j in range( v_num ):
+            if i > j:
+                e[0][idx] = i
+                e[1][idx] = j
```

Comment:
For NumPy arrays it is usual (and more efficient) to index like:

```suggestion
                e[0, idx] = i
                e[1, idx] = j
```

Reply:
Done